### PR TITLE
Update om-statistikk.no.mdx

### PR DESCRIPTION
**Pull request by Caroline-A**

### CODE REVIEW

File: pages/statistikk/om-statistikk.no.mdx [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+
 # Statistikk
 
-Statistikk om SDG-forskning i Norge er under arbeid, og blir lansert etter at datasettet er komplett (estimert tidlig 2023). 
\ No newline at end of file
+Statistikk om SDG-forskning i Norge er under arbeid, og blir lansert etter at datasettet er komplett (estimert tidlig 2023). 
+
+<Tableau 
+  hostUrl='https://rapport-dv.uhad.no/t/DUCT/views/SDG_statistics_2023-05/Dashboard1' 
+  height='640'
```
Comment (Caroline-A) [inline pages/statistikk/om-statistikk.no.mdx:9]: Must be set, or?

Comment (tarjelavik) [reply pages/statistikk/om-statistikk.no.mdx]: No, but if not set it will be a bit too tall.

**Comment by Caroline-A** [comment]
@tarjelavik Is the height set at 580 because (reasons) or was it relatively random? I'm wondering if I should adjust the structure of the figures to this height or if it is ok to increase the height of the frame

**Comment by tarjelavik** [comment]
I think it need a height. But we can test.

File: components/tableau/index.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const TableauNoSSR = dynamic(() => import('@stoddabr/react-tableau-embed-live').
 export const Tableau = (
   { hostUrl, height, width = '100%', toolbar = 'hidden', device = 'default' }: { hostUrl: string, height?: number | string, width?: number | string, toolbar?: 'top' | 'bottom' | 'hidden', device?: 'default' | 'desktop' | 'tablet' | 'phone' }) => {
   return (
-    <div className={`mx-auto w-full bg-slate-100 rounded p-1 my-3 overflow-scroll`}>
+    <div className={`mx-auto w-full bg-slate-100 rounded p-1 my-3 overflow-auto`}>
```
Comment (Caroline-A) [inline components/tableau/index.tsx:13]: I think overflow-scroll looked ok on mac as it hides the scroll bars when not needed, but on other systems it forces them to show (https://tailwindcss.com/docs/overflow#scrolling-in-all-directions). So have changed to auto - has got rid of them on my phone/pc at least.

**Comment by Caroline-A** [comment]
- Norwegian text checked by Eli - ok
- Waiting until next data update (SDG13++) - remember to include 2022 & English versions then

**Comment by Caroline-A** [comment]
This one is now ready for merging @tarjelavik if you could do that and sort the conflicts? Thanks! 🦦

**Pull request merged**